.github/workflows/run_tests.yml

```diff
@@ -118,6 +118,16 @@ jobs:
         if: (matrix.py_v == '3.10')
         run: smart build --device cpu -v
 
+      - name: Run mypy
+        # TF 2.6.2 has a dep conflict with new mypy versions
+        if: (matrix.rai != '1.2.5')
+        run: |
+          python -m pip install .[mypy]
+          make check-mypy
+
+      - name: Run Pylint
+        run: make check-lint
+
```

Comment:
You might need to update the changes to the yml file here based on what happened in SmartRedis

Reply:
Fixed!